synapseclient/client.py

```diff
@@ -2313,6 +2314,28 @@ def getPermissions(
                 )
         return list(effective_permission_set)
 
+    @tracer.start_as_current_span("Synapse::get_permissions")
+    def get_permissions(
+        self, entity: Union[Entity, Evaluation, str, collections.abc.Mapping]
+    ):
+        """Get the permissions that the caller has on an Entity.
+        https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/auth/UserEntityPermissions.html
+
+        :param entity:      An Entity or Synapse ID to lookup
```

Comment:
Let's swap to using google style docstrings as that's what we want to use moving forward.

https://github.com/Sage-Bionetworks/synapsePythonClient/blob/a8bc6a899bfda682c998043bd5f813386dd9c1e6/synapseclient/client.py

For an example of what it looks like for the new doc pages.

In VSC my `Auto Docstring: Docstring Format` is set to:
`google-notypes`